repository/get_todo_repository.go

```diff
@@ -0,0 +1,29 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/jinzhu/copier"
+	"todo-clean/common"
+	"todo-clean/domain"
+	"todo-clean/lib/error_lib"
+	"todo-clean/repository/model"
+)
+
+func (repo newRepo) GetTodoRepository(ctx context.Context, key uuid.UUID) (resp []domain.GetTodoEntity, err error) {
```

Comment:
Note: It is better to rename `resp` to `results` for getting more descriptive name